test/e2e/testsuites/dynamically_provisioned_pod_failover_tester.go

```diff
@@ -0,0 +1,79 @@
+/*
+Copyright 2020 The Kubernetes Authors.
```

Comment:
@kassarl It appears the `pull-azuredisk-csi-driver-verify` test case is failing because the boilerplate header is missing the expected blank lines. See https://github.com/kubernetes-sigs/azuredisk-csi-driver/blob/master/hack/boilerplate/boilerplate.go.txt for the expected format (replacing 'YEAR' with the numeric year).